package.json

```diff
@@ -80,7 +80,7 @@
     "typescript": "^4.9.4"
   },
   "jest": {
-    "preset": "jest-expo",
```

Comment:
By default jest-expo runs in a mock ios env. So switched it to android, but doubt it would matter for the types of testing we are doing.